README.md

```diff
@@ -60,6 +60,7 @@ nufmt my-file1.nu my-file2.nu my-file3.nu
 
 ### Options
 
+- `--check` if you just want to check files without formatting them. It cannot be combined with stdin.
```

Comment:
--check is typically called --dry-run in cli's. i can live with either though.

Reply:
I will have a look at other code formatters, see what they use

Reply:
I also like --dry-run, but it looks like a lot of formatters use --check:
- **Use --check**: black, ruff, prettier, rustfmt
- **Use --dry-run**: clang-format, gofmt

Both are fine by me, although maybe --dry-run could make the code easier.
What do you think?

Reply:
@AucaCoyan any preference?

Reply:
No preference at all! both are also used in other CLI tools (non-formatters) so both ways are good for me!

Whatever suits you

Reply:
I replaced --check by --dry-run. I had a slight preference for this one. Thanks for the suggestion @fdncred !